vic_run/src/runoff.c

```diff
@@ -39,48 +39,51 @@ runoff(cell_data_struct  *cell,
        soil_con_struct   *soil_con,
        double             ppt,
        double            *frost_fract,
-       int                dt,
+       double             dt,
```

Comment:
It appears to be OK.  I assume that somewhere there is a check to ensure that the runoff time step length is never greater than the model time step length?